Updates README to be a bit more descriptive (#220)
DAGWorks-Inc/burr: README.md

<a href="https://discord.gg/6Zy2DwP4f3" target="_blank"><img src="https://img.shields.io/badge/Join-Burr_Discord-brightgreen?logo=discord" alt="Burr Discord"/></a>
</div>

> Burr makes it easy to develop applications that make decisions based on state (chatbots, agents, simulations, etc...) from simple python building blocks. It is particularly well-suited towards anything that leverages LLMs. Unlike other frameworks, however, it does not dictate how you craft prompts, integrate AI, or build out your user interfaces. Rather, it provides the way to do that in a robust, trackable manner, using whatever frameworks you want. Burr includes a UI that can track/monitor those decisions in real time.

Author: ```suggestion
Burr makes it easy to develop applications that make decisions based on state (chatbots, agents, simulations, etc...) from simple python building blocks. 

It is particularly well-suited towards anything that leverages LLMs. Burr does not dictate how you craft prompts, integrate AI, or build out your user interfaces. Rather, it provides tools to do that in a robust, trackable manner, using whatever frameworks you want. Burr includes a UI that can track/monitor those decisions in real time.
```
